CODEOWNERS

```diff
@@ -1 +1,4 @@
-# Add the list of code owners here (using their GitHub username)
+AlexandruCihodaru
```

Comment:
should this part follow github's standard? You may need an @ before each github username
(https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)